sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1141,7 +1141,7 @@ pi_result resetCommandLists(pi_queue Queue) {
          it != Queue->CommandListMap.end(); ++it) {
       // It is possible that the fence was already noted as signalled and
       // reset. In that case the InUse flag will be false.
-      if (it->second.InUse) {
+      if (it->second.ZeFence != nullptr && it->second.InUse) {
```

Comment:
This change is good, but I suggest to generalize it:
1) rename InUse to ZeFenceInUse
2) fix few other places where inUse was used not under ZeFence != nullptr check

Reply:
Thanks for the feedback @smaslov-intel. I have addressed this in the current commit. Please verify.